frontend/src/shared/helpers.tsx

```diff
@@ -433,56 +438,58 @@ const readOnlyHeader = {
   `,
 };
 
-SampleDetailsColumns.forEach((colDef) => {
-  colDef.cellClassRules = {
-    unsubmittedChange: (params: any) => {
-      const changes = params.context.getChanges();
-      const changedValue = changes?.find((change: any) => {
-        return (
-          change.fieldName === params.colDef.field &&
-          change.primaryId === params.data.primaryId
-        );
-      });
-      return changedValue !== undefined;
-    },
-  };
+function setupEditableSampleFields(samplesColDefs: ColDef[]) {
```

Comment:
The codes below used to be applied to just the regular, non-Cohort Sample views to support the edit-ability of fields. Now that Cohort Sample view is also editable, I put these codes into a named function (`setupEditableSampleFields`) so that we can apply them to all sample views.